src/transformStateWithClones.js

```diff
@@ -7,7 +7,33 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const resultObject = { ...state };
+
+  const resultArray = [];
+
+  for (const action of actions) {
+    if (action.type === 'clear') {
```

Comment:
Use destruction for `action`
```suggestion
  for (const action of actions) {
    const { type, ....... } = action;
  
    if (type === 'clear') {
```